src/__tests__/md-inject.test.ts

```diff
@@ -312,6 +319,60 @@ The output of some arbitrary command
     expect(fs.writeFile).toHaveBeenCalledWith('foo.md', outFile)
   })
 
+  it('writes to the markdown document (command) with mdx syntax', async () => {
+    mock({
+      mockFileName: 'foo.mdx',
+      config: {
+        type: 'command',
+        value: 'some arbitrary command',
+      },
+      mockResponse: 'The output of some arbitrary command',
+    })
+
+    await injectMarkdown()
+
+    const outFile = `
+{/* CODEBLOCK_START {"type":"command","value":"some arbitrary command"} */}
+<!-- prettier-ignore -->
```

Comment:
building on @ckemp1's excellent review, I think this is also a stray HTML style comment within an MDX file, and it's presence here (and passing test) would indicate that this is perhaps still not working? I didn't run it locally yet, but wanted to note this.

Reply:
We've been chatting back and forth, and that issue should be resolved now. All there is left to tackle would be my first concern when running the script itself
>We want the output to also use the same mdx comment syntax, but instead it seems to be still adding in html styled comments

Reply:
This issue has now been resolved and verified and is explained in the PR body